cpp/test/Ice/metrics/AllTests.cpp

```diff
@@ -610,38 +610,6 @@ allTests(Test::TestHelper* helper, const CommunicatorObserverIPtr& obsv)
 
         metrics->ice_getConnection()->close(Ice::ConnectionClose::GracefullyWithWait);
 
-        metrics->ice_timeout(500)->ice_ping();
```

Comment:
I would be good to add back a metrics test that tests connection failures. It's not clear to me how to do it. I'll create an issue .